test_workspacebuilder.py won't finish - no error message

### Step 1: Provide a summary of your problem

I was trying to run the tests with `make test` and the workspacebuilder_test seems to run endlessly (did have it running for more than 10 minutes, 2 times) before exiting with C-c. 

Sorry if that is a bit too unspecific but I am not that expierienced with tests. So I'll gladly take advice. The reason for running the tests is that I am working on an issue / feature request.

Edit: okay I narrowed it down to the `def pane_order_test()`. I marked it to be skipped. All other tests pass.
Edit2: I narrowed it down further: the while loop in the test, at least for me, leads to an infinite loop because the breaking condition of `if p.current_path == pane_path` is never met, it always compares `/usr/bin` vs. `/usr/sbin`.

### Step 2: Provide tmuxp details
  * Python version 3.8.3
  * system PATH 
  * tmux version 3.1b
  * tmuxp version 1.5.5 
  * tmux path 
  * tmuxp path home/arkedos/Documents/git/tmuxp-dev/bin/tmuxp (virtualenv)
  * libtmux version 0.8.2
  * shell - zsh

### Step 3: Describe your environment
  * Architecture: x86_64
  * OS version: Manjaro Linux 20.2 Kernel 5.8 RC

### Step 4: Describe the problem:
#### Steps to reproduce:
  1. run `make test`
  2. wait...
  3. hit C-c to exit after 10+ Minutes

#### Observed Results:
  * What happened?  This could be a description, log output, etc.
Not sure what to provide here since I am not that expierenced with tests. So advice is needed.

#### Expected Results:
  * What did you expect to happen?
To finish all tests.

#### Relevant Code:
Did not make changes to testing code.

It seems like the libtmux module doesn't have `return_until` yet:

```text
E   ImportError: cannot import name 'retry_until' from 'libtmux.test' (/usr/lib/python3.10/site-packages/libtmux/test.py)
```
I am using `python-libtmux 0.11.0`.

--- a/tests/fixtures/workspacebuilder/pane_ordering.yaml
+++ b/tests/fixtures/workspacebuilder/pane_ordering.yaml
@@ -7,5 +7,5 @@ windows:
   panes:
   - cd /usr/bin
   - cd /usr
-  - cd /usr/sbin
+  - cd /etc
   - cd {HOME}

--- a/tests/test_workspacebuilder.py
+++ b/tests/test_workspacebuilder.py
@@ -525,7 +525,7 @@ def test_pane_order(session):
     pane_paths = [
         "/usr/bin",
         "/usr",
-        "/usr/sbin",
+        "/etc",
         os.path.realpath(os.path.expanduser("~")),
     ]
 
@@ -539,10 +539,10 @@ def test_pane_order(session):
 
     window_count = len(session._windows)  # current window count
     assert len(s._windows) == window_count
+
     for w, wconf in builder.iter_create_windows(s):
         for p in builder.iter_create_panes(w, wconf):
             w.select_layout("tiled")  # fix glitch with pane size
-            p = p
             assert len(s._windows) == window_count
 
         assert isinstance(w, Window)
@@ -563,9 +563,7 @@ def test_pane_order(session):
                 p.server._update_panes()
                 return p.current_path == pane_path
 
-            retry_until(f)
-
-            assert p.current_path, pane_path
+            assert retry_until(f)
 
 
 def test_window_index(session):

